code/game/machinery/doors/door.dm

```diff
@@ -347,27 +347,8 @@
 		open()
 
 /obj/machinery/door/proc/crush()
```

Comment:
/obj/machinery/door/proc/crush() why are you deleting this proc pretty much and then overriding it in both airlock and firelocks child just leave it in there and override the child for firelocks only.

Reply:
the parent proc will not get called unless you use . = ..() to actually call it

Reply:
Reason why i am saying this is cause i am pretty sure shutters too are a child of doors so you deleting and replacing this parent proc with return would break crush on shutters.

Reply:
yeah this is the obj path of shutters /obj/machinery/door/poddoor/shutters if you do it like this shutters when they call crush would only get null from return.

Reply:
fuck

Reply:
aite